packages/api-server/src/index.ts

```diff
@@ -23,3 +28,5 @@ http({ port, socket }).on('listening', () => {
   }
   console.log()
 })
+
+export { routePrefix }
```

Comment:
Should this be exported?
```suggestion
```

Reply:
I resolve this in favor of commenting the next commit.